packages/grid/_modules_/grid/models/gridSlotsComponent.ts

```diff
@@ -96,6 +105,7 @@ export const DEFAULT_SLOTS_COMPONENTS: ApiRefComponentsProperty = {
   ErrorOverlay,
   Footer: GridFooter,
   Header: GridHeader,
+  PreferencesPanel,
```

Comment:
```suggestion
  PreferencesPanel: GridPreferencesPanel
```

Reply:
I can rename it, not a problem. I saw that there are other components that don't have the `Grid` suffix. If agree that this will be what we will do from now one I'm ok with it.

Reply:
We treat all the exported modules of Material-UI as a unique namespace, as AWS does with its code. Developers should be able to search for it without landing on potential different components. If there a chance that the component might collide, with another, it needs to be prefixed for uniqueness. This applies to even the types.

Reply:
Ok, but that means that some of the others (maybe even all) that are here https://github.com/mui-org/material-ui-x/blob/master/packages/grid/_modules_/grid/models/gridSlotsComponent.ts#L99-L104 would need to be renamed.

Reply:
Ok, let's do it in a different effort. We have https://github.com/mui-org/material-ui-x/issues/131 to track it.

Reply:
I have made the same comment in https://github.com/mui-org/material-ui-x/pull/969#pullrequestreview-580841832.